AuroraEditor/Core/InspectorSidebar/UI/History/HistoryItem.swift

```diff
@@ -87,35 +87,35 @@ struct HistoryItem: View {
                 Button("Copy Commit Message") {
                     let pasteboard = NSPasteboard.general
                     pasteboard.clearContents()
-                    pasteboard.setString(commit.message, forType: .string)
+                    pasteboard.setString(commit.summary, forType: .string)
                 }
                 Button("Copy Identifier") {}
                     .disabled(true) // TODO: Implementation Needed
-                Button("Email \(commit.author)...") {
+                Button("Email \(commit.author.name)...") {
                     let service = NSSharingService(named: NSSharingService.Name.composeEmail)
-                    service?.recipients = [commit.authorEmail]
+                    service?.recipients = [commit.author.email]
                     service?.perform(withItems: [])
                 }
                 Divider()
             }
             Group {
-                Button("Tag \(commit.hash)...") {}
+                Button("Tag \(commit.shortSha)...") {}
                     .disabled(true) // TODO: Implementation Needed
-                Button("New Branch from \(commit.hash)...") {}
+                Button("New Branch from \(commit.shortSha)...") {}
                     .disabled(true) // TODO: Implementation Needed
-                Button("Cherry-Pick \(commit.hash)...") {}
+                Button("Cherry-Pick \(commit.shortSha)...") {}
                     .disabled(true) // TODO: Implementation Needed
             }
             Group {
                 Divider()
-                if let commitRemoteURL = commit.commitBaseURL?.absoluteString {
-                    Button("View on \(commit.remoteString)...") {
-                        let commitURL = "\(commitRemoteURL)/\(commit.commitHash)"
-                        openCommit(URL(string: commitURL) ?? emptyURL)
-                    }
-                    Divider()
-                }
-                Button("Check Out \(commit.hash)...") {}
+//                if let commitRemoteURL = commit.?.absoluteString {
+//                    Button("View on \(commit.remoteString)...") {
+//                        let commitURL = "\(commitRemoteURL)/\(commit.commitHash)"
+//                        openCommit(URL(string: commitURL) ?? emptyURL)
+//                    }
+//                    Divider()
+//                }
```

Comment:
Is this deprecated / not used anymore?

Reply:
No just commented out till i find a better way of doing it with the apis